x-pack/plugins/monitoring/server/deprecations.ts

```diff
@@ -22,6 +23,12 @@ export const deprecations = ({
 }: ConfigDeprecationFactory): ConfigDeprecation[] => {
   return [
     deprecate('enabled', '8.0.0', { level: 'critical' }),
+    deprecate('cluster_alerts.allowedSpaces', '8.0.0', {
+      level: 'warning',
+      message: i18n.translate('xpack.monitoring.deprecations.allowedSpaces', {
+        defaultMessage: 'Cluster alerts are now created after opt-in.',
```

Comment:
```suggestion
        defaultMessage:
          'Starting in 7.15, the Stack Monitoring application will prompt you to create cluster alerts rather than creating them automatically in allowed spaces.' +
          ' This configuration is no longer used and will be removed in a future version. ',
```

Reply:
That's much clearer, I'll update